packages/app-mobile/components/checkbox.tsx

```diff
@@ -13,33 +27,36 @@ const styles = {
 };
 
 class Checkbox extends Component {
```

Comment:
```suggestion
class Checkbox extends Component<Props, State> {
```
See [creating class `Component`s with TypeScript](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/class_components/).

Reply:
Thanks for this.